sfge/src/test/java/com/salesforce/rules/unusedmethod/BaseUnusedMethodTest.java

```diff
@@ -260,11 +259,16 @@ protected void assertExpectations(
             Collection<String> usedMethodKeys,
             Collection<String> unusedMethodKeys) {
 
-        //TODO: This method attempts to simulate what AbstractRuleRunner.runRules does minus all the thread specific code
-        //      We should refactor the AbstractRuleRunner so that the Thread Specific parts can be injected as a strategy
-        //      and then for testing we would inject a strategy that executes without threads so that we can simply
-        //      call through to the top level rule runner here in the tests. Doing this refactoring would help us
-        //      prevent accidentally forgetting to update this code when changes occur somewhere deep down
+        // TODO: This method attempts to simulate what AbstractRuleRunner.runRules does minus all
+        // the thread specific code
+        //      We should refactor the AbstractRuleRunner so that the Thread Specific parts can be
+        // injected as a strategy
+        //      and then for testing we would inject a strategy that executes without threads so
+        // that we can simply
+        //      call through to the top level rule runner here in the tests. Doing this refactoring
+        // would help us
```

Comment:
Look at this? So ugly. This is "spotless" killing the formatting because for some reason even though the default for all IDEs is to have 120 line width... spotless is using 100 or something as default. If you look at the diff between my two commits in this review you'll see the before and after of spotless.

This may help a little:
		prettier(mapOf("prettier" to "2.8.8", "prettier-plugin-java" to "2.5.0"))
			.config(mapOf("parser" to "java", "tabWidth" to 4, "printWidth" to 120))
			
But to be honest... I'd love to get rid of this forced formatting. I think human's are smarter than computers when it comes to what is most readable with formatting.